lib/virt_autotest/utils.pm

```diff
@@ -547,7 +547,8 @@ sub remove_additional_nic {
 
 sub collect_virt_system_logs {
     if (script_run("test -f /var/log/libvirt/*d.log") == 0) {
-        upload_logs("/var/log/libvirt/*d.log");
+        script_run('tar czvf /tmp/libvirt_daemons.tar.gz /var/log/libvirt/*d.log');
```

Comment:
Default timeout enough for generate tar ball if large and many log files ?

Reply:
I just made a search that most of 'tar' runs without a timeout option. I think it is ok in most cases.
The function is used by MU tests and I am unable to tell or verify a clear result. Let MU testers handle it if timeout happens in the future.